packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx

```diff
@@ -37,24 +35,14 @@ interface TreeNodeRendererOwnProps {
   reloadTree?: (options: { parentNodeId: string | undefined; state: "reset" }) => void;
   /** CSS class name for the action buttons. */
   actionButtonsClassName?: string;
```

Comment:
if we allow consumers to render anything through `actionsRenderer`, does `actionButtonsClassName` have any value anymore?

Reply:
That was removed.